discussment-ws/docs/discussment-service-v1.yaml

```diff
@@ -339,7 +339,7 @@ types: !include discussment-service-v1-types.yaml
             description: Reply added successfully.
             body: PostDTO
           400:
-            description: Maximum level of replies added to a post exceeded.
+            description: Maximum level of replies added to a post exceeded or the reply text is too long.
```

Comment:
Can you regenerate HTML as well? We use [raml2html](https://www.npmjs.com/package/raml2html) tool for it.

Usage: `raml2html discussment-service-v1.yaml > discussment-service-v1.html`